.github/workflows/lint.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           pip install -q -r requirements.txt
           pip install -q mypy mypy-extensions
           pip install -q git+https://github.com/numpy/numpy-stubs.git
-          pip install -q torch==1.5.1+cpu torchvision==0.6.1+cpu -f https://download.pytorch.org/whl/torch_stable.html
+          pip install -q torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
```

Comment:
we may want to pin this to a torch version (2.0.0?)

Reply:
I tried copying directly from PyTorch Install instruction. This seems to be the recommended way now. We could either mend it if/when it broke later, or have it as is. The problem with pinning to a special version has been haunting our tests for a long time. Let's see what happens with keeping them to  lower level for a while?